agave_pyclient/agave_pyclient/agave.py

```diff
@@ -173,6 +173,8 @@ class AgaveRenderer:
     def __init__(self, url="ws://localhost:1235/", mode="pathtrace") -> None:
         self.cb = CommandBuffer()
         self.session_name = ""
+        if mode != "pathtrace" and mode != "raymarch":
+            mode = "pathtrace"
```

Comment:
bonus change: the python client will fix up the mode string if user supplied an invalid one.